src/test/java/reposense/parser/CsvParserTest.java

```diff
@@ -173,6 +182,12 @@ public void authorConfig_emptyLocation_success() throws ParseException, IOExcept
         Assert.assertEquals(AUTHOR_CONFIG_NO_SPECIAL_CHARACTER_AUTHORS, authorConfig.getAuthorList());
     }
 
+    @Test (expected = IOException.class)
+    public void authorConfig_emptyConfig_fail() throws IOException {
```

Comment:
For method name, instead of just stating `fail`, better to state a better descriptive result (`..._throwsIoException()`)

Reply:
Done.